src/include/ndpi_typedefs.h

```diff
@@ -775,6 +775,7 @@ struct ndpi_flow_udp_struct {
 
   /* NDPI_PROTOCOL_SKYPE */
   u_int8_t skype_packet_id;
+  char skype_crc[4];
```

Comment:
An `u_int8_t` aka `unsigned char` would fit better as it indicates a non-printable byte buffer.

Reply:
Fixed.